draft-jholland-quic-multicast.md

```diff
@@ -212,6 +212,11 @@ Note that the hash is on the unencrypted packet because it checks against a spec
 
 # Packet Scheduling
 
+# Stateless Reset
+
+As clients can unilaterally stop the delivery of multicast packets by leaving the relevant Group or Channel, sessions do not need stateless reset tokens.
+Instead, if an endpoint receives a packet addressed to a multicast group it can not associate with an existing session, it SHOULD take the necessary steps to prevent the reception of further such packets. For example, it might issue an IGMP or MLD report indicating a desire to leave the (S,G) associated with the unwanted packet.
```

Comment:
I think this might be too strong.  An attack that injects a packet with a bad session id could cause everyone to leave.

That said, if there's a lot of traffic with an unknown session id arriving on a network path, I do think it's appropriate to leave, perhaps with a MAY, or with a SHOULD if it's above some threshold (not sure if we need it, but server could perhaps set a recommended threshold in session properties if we do?).  But I think this should be less about issuing an IGMP membership report and more like "the client leaves the session(s) corresponding to the network path receiving excessive traffic with unknown session IDs with a leave reason of 'excessive spurious traffic'."

Reply:
I agree with the first part, good point thanks. I think it might be ok to say that it is the Group Address that can't be associated with any channel since the Endpoint should know all group addresses it expects traffic on. 

As for the second part, as far as I understand stateless resets, it is for cases where the client loses all state so it might not know which the corresponding unicast connection is or it might lose that connection all together as well. I think the point is to just stop the sending/forwarding/receiving of useless data to not expand resources unnecessarily. (I think that's why stateless reset token can be generated from scratch if the only thing known is the connection ID for which to issue the stateless reset token).